### ISSUE TITLE
Support for Forward+ rendering with GPU resident drawer/ GPU occlusion for Unity 6

### ISSUE THREAD

**Issue by Squashi11** [oculus-samples/Unity-DepthAPI#68]
Do you ever plan on implementing support for Forward+ with GPU resident drawer/ GPU occlusion in Unity 6? It could come in handy for performance optimizations in mixed reality. For now it throws some errors and refuses to display some materials. But the performance is improved by 10-20% in my scenes. Thanks for the reply
![image](https://github.com/user-attachments/assets/7ea62c7d-e790-4ea7-a210-9fd80c563bd3)

**Comment by TudorJude** [comment]
Hello,

We've found the issue behind this. Any shader that breaks batching will not work. This became apparent when I set "BatchRendererGroup" to "Keep All". The shader that we provide breaks batching because of how we implemented the "Environment Depth Bias" uniform, we should have added it to the "UnityPerMaterial" CBuffer. We'll update the packages in this github to remedy this and update this thread once they are up.

Thank you for discovering this issue.
